frontend/src/components/UniversalLandingPageButton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styles from '../styles/UniversalLandingPageButton.module.scss';
+import { Link } from 'react-router-dom';
+
+
+
+function UniversalLandingPageButton(){
```

Comment:
Since you are setting specific width and height for PLAYER and HOST buttons, do they have to be a component? A component is meant to be reused. Since these buttons are not reused, it's best to but them in the same UniversalLandingPage component for the sake of readability